app/models/miq_server/worker_management/monitor/start.rb

```diff
@@ -10,7 +10,8 @@ def wait_for_started_workers
     entered_wait_for_started_loop = Time.now.utc
     wait_for_started_timeout      = @worker_monitor_settings[:wait_for_started_timeout] || 10.minutes
     loop do
-      starting = MiqWorker.find_starting.find_all { |w| MiqWorkerType.worker_class_names.include?(w.class.name) }
+      starting = MiqWorker.find_starting.find_all { |w| MiqWorkerType.worker_class_names.include?(w.class.name) }.to_a
+      sync_starting_workers!(starting)
```

Comment:
We'll need to add a `sync_starting_workers!` method to the Process and Kubernetes subclasses as well, even if they are empty because this will fail on those.

Reply:
We should probably add an abstract method to the base class also.